recipes/libsndfile/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "libsndfile" %}
+{% set version = "1.0.27" %}
+{% set md5 = "fd1d97c6077f03b5d984d7956ffedb7a" %}
```

Comment:
conda-forge prefers sha256. In this case `a391952f27f4a92ceb2b4c06493ac107896ed6c76be9a613a4731f076d30fac0`